input.go

```diff
@@ -2,10 +2,11 @@ package main
 
 import (
 	"fmt"
+	"strings"
 	"time"
 	"unicode/utf8"
 
-	"github.com/japanoise/termbox-util"
+	termutil "github.com/japanoise/termbox-util"
```

Comment:
Comment: Modern go import style. Amazed GNU Emacs' go-mode didn't catch this years ago. I'm not particularly fussed about this being a clean commit though, so no worries.

Reply:
> Comment: Modern go import style. Amazed GNU Emacs' go-mode didn't catch this years ago. I'm not particularly fussed about this being a clean commit though, so no worries.

I [have](https://github.com/ixtenu/rc/blob/b51162ed82dcffdf6fdd62cd82a962032e2b77de/.emacs.d/init.el#L503) a save hook which runs `lsp-organize-imports` (with `gopls` as the LSP backend).  That's where this change comes from.  If commit cleanliness is an issue, I could turn that off (or just run it on all the \*.go files and submit a PR with only those changes).

Reply:
> > Comment: Modern go import style. Amazed GNU Emacs' go-mode didn't catch this years ago. I'm not particularly fussed about this being a clean commit though, so no worries.
> 
> I [have](https://github.com/ixtenu/rc/blob/b51162ed82dcffdf6fdd62cd82a962032e2b77de/.emacs.d/init.el#L503) a save hook which runs `lsp-organize-imports` (with `gopls` as the LSP backend). That's where this change comes from. If commit cleanliness is an issue, I could turn that off (or just run it on all the *.go files and submit a PR with only those changes).

It will eventually come up in other changes, I'm sure, so yes, please go ahead and run it on all the go files (except bindata) and make a change for that. This way is better anyway as it makes it more clear what termbox-util's package is called (a boneheaded decision of mine from back when termutil had just two or three functions).